src/panels/lovelace/cards/hui-sensor-card.ts

```diff
@@ -90,7 +90,7 @@ const calcPoints = (
   let last = [average(first), lastValue(first)];
 
   const getCoords = (item, i, offset = 0, depth = 1) => {
-    if (depth > 1) {
+    if (depth > 1 && item) {
```

Comment:
Shouldn't we instead guard `getCoords` from being called on `undefined` ? What other things will break?

Reply:
The reason that getCoords should run on undefined is to get the smooth line as getCoords uses the last point to plot the value if item is undefined. If we dont call it we get this 
![image](https://user-images.githubusercontent.com/18730868/74606848-d31be980-50a1-11ea-87d0-d706cf229650.png)